client-java/controller/src/test/java/org/evomaster/clientJava/controller/internal/db/TableDtoUtils.java

```diff
@@ -0,0 +1,29 @@
+package org.evomaster.clientJava.controller.internal.db;
+
+import org.evomaster.clientJava.controllerApi.dto.database.schema.TableDto;
+
+import java.util.List;
+
+public class TableDtoUtils {
+
+
+    public static boolean containsTable(List<TableDto> tables, String tableName) {
+        for (TableDto tableDto :
+                tables) {
```

Comment:
why the new line for just "table"?

Reply:
the code-autocompletion played a trick on me. I will fix it.